share/man/man9/vmem.9

```diff
@@ -74,7 +77,7 @@ other than virtual memory.
 .\" - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 .Fn vmem_create
 creates a new vmem arena.
-.Bl -tag -width qcache_max
+.Bl -tag -width "qcache_max"
```

Comment:
```suggestion
.Bl -tag -offset indent -width "qcache_max"
```

The meta-alignment of the lists is fine; I don't have a strong opinion either way.

I think it is important is to add an `-offset` to each of these top-level lists. This does more to improve the overall legibility of the document.